core/src/main/scala/scalaz/LazyEitherT.scala

```diff
@@ -189,6 +189,12 @@ sealed abstract class LazyEitherTInstances1 {
       def iso = LazyEitherT.lazyEitherTLeftProjectionEIso2[F, L]
     }
 
+  implicit def lazyEitherTBindRec[F[_], L](implicit F0: Monad[F], B0: BindRec[F]): BindRec[LazyEitherT[F, L, ?]] =
```

Comment:
Seems like this will be ambiguous with the next implicit when looking for:

```
Bind[LazyEitherT[F, ?, ?]]
```

Given `F: BindRec`.